### DISCUSSION TITLE
📊 Vote for the tools or libraries you would like to see ported over to z/OS

### DISCUSSION THREAD

**Discussion by IgorTodorovskiIBM** [zopencommunity/meta#298] [Announcements]
📊 [Vote here](https://app.sli.do/event/wMWtLJmrTSC5Gfg3sxmXyf/live/polls)

Your feedback will help us to prioritize our future efforts! Thanks!

**Comment by gngrossi** [comment]
Are other tools being included in coreutils? If so, which ones are planned?
Some suggestions:  groups, shred, whereis 

Suggested language:  Rust

thanks

**Reply by IgorTodorovskiIBM** [reply]
Thanks Gary,
I just gave `groups` and `shred` a try and they're working. I'll check the tests, if they're passing, we can expose them.

`whereis` appears to be part of util-linux, I'll add that to the poll

**Comment by IgorTodorovskiIBM** [comment]
Here are the latest poll results as of June 12:
[2Q_Poll_zOS_Open_Tools.pdf](https://github.com/ZOSOpenTools/meta/files/11725072/2Q_Poll_zOS_Open_Tools.pdf)

**Reply by v1gnesh** [reply]
Like the quote from Henry Ford - "If I had asked people what they wanted, they would have said faster horses."
The fact that Arrow, eBPF, WireGuard aren't known/desired is depressing.
fzf too... it's brilliant; seeing it at 0% is sad.

Although it's fun to see Rust's popularity in the poll, ripgrep geting 0% makes me wonder how much of the "demand" is because Rust is a cool thing to want these days...

Will be nice to have newer OpenSSH, as it comes with the ability to use SSH keys for Git commit signing.

**Reply by IgorTodorovskiIBM** [reply]
Agreed, I think part of the problem is that many of the tools that got 0 votes are foreign to the poll audience (z/OS folks).  Perhaps future polls can provide descriptions of the tools along with their respective benefits.

**Reply by v1gnesh** [reply]
> I think part of the problem is that many of the tools that got 0 votes are foreign to the poll audience (z/OS folks). 

That's exactly why ZOSOpenTools is critical to IBM Z and zOS's future.
It's an opportunity to lead 'da way' by bringing the best of distributed to Z.
Shouldn't let it settle for "faster horses" :)

**Comment by IgorTodorovskiIBM** [comment]
Closing this poll in favour of the new one.
